pkg/healthmonitor/metrics.go

```diff
@@ -0,0 +1,64 @@
+package healthmonitor
+
+import (
+	compbasemetrics "k8s.io/component-base/metrics"
+)
+
+type registerables []compbasemetrics.Registerable
+
+var (
+	healthyTargetsTotal = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "health_monitor_healthy_target_total",
+			Help:           "Number of healthy instances registered with the health monitor. Partitioned by targets",
```

Comment:
this should be a gauge, no?

Reply:
we could have a gauge to track the current number of healthy targets, alert when is `0` for some time.